src/app/record/pages/my-orcid/my-orcid.component.ts

```diff
@@ -66,6 +63,7 @@ export class MyOrcidComponent implements OnInit, OnDestroy {
       .subscribe((userRecord) => {
         this.recordWithIssues = userRecord?.userInfo?.RECORD_WITH_ISSUES
         this.userRecord = userRecord
+        this.loading = true
```

Comment:
The PR https://github.com/ORCID/ORCID-Source/pull/6241 it is necessary to have the PR merged since the current code is throwing a 500 and the server won't reply and `loading` won't change to true when a user is not found

Reply:
we are going to merge this on the next QA release, after the PROD release according to angel